ci-operator/jobs/openshift/assisted-image-service/openshift-assisted-image-service-main-presubmits.yaml

```
@@ -1,5 +1,158 @@
 presubmits:
   openshift/assisted-image-service:
+  - agent: kubernetes
+    always_run: true
```

seems like it regressed: 
```suggestion
    always_run: false
```